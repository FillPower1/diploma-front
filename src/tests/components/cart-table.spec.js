import React from 'react'
import { shallow } from 'enzyme'
import CartTable from '../../components/cart-table'

describe('Cart Table', () => {
	const props = {
		items: []
	}

	describe('Cart Table render', () => {

		const nextProps = {
			...props,
			items: [{ id: 23, 'title': 'title', 'price': 25 }]
		}

		const cartTable = shallow(<CartTable {...nextProps} />)

		it('renders properly', () => {
			expect(cartTable).toMatchSnapshot()
		})

		it('render CartItem', () => {
			expect(cartTable.find('CartItem')).toHaveLength(1)
		})
		
		it('render ProductPrice', () => {
			expect(cartTable.find('ProductPrice')).toHaveLength(1)
		})
	})
})