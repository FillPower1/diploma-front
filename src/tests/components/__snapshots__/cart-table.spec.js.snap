// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart Table Cart Table render renders properly 1`] = `ShallowWrapper {}`;
